tracer/test/snapshots/Couchbase3Tests.verified.txt

```diff
@@ -0,0 +1,293 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: couchbase.query,
+    Resource: Delete,
+    Service: Samples.Couchbase3-couchbase,
+    Type: db,
+    Tags: {
+      component: Couchbase,
+      couchbase.operation.code: Delete,
+      couchbase.operation.key: my-document-key,
+      env: integration_tests,
+      runtime-id: Guid_1,
+      span.kind: client,
+      _dd.p.dm: -0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_3,
+    SpanId: Id_4,
+    Name: couchbase.query,
+    Resource: Delete,
+    Service: Samples.Couchbase3-couchbase,
+    Type: db,
+    Error: 1,
+    Tags: {
+      component: Couchbase,
+      couchbase.operation.code: Delete,
+      couchbase.operation.key: does-not-exist,
+      env: integration_tests,
+      error.type: KeyNotFound,
```

Comment:
I don't understand this one. In other snapshots, we have an exception. I believe your code is good,  so does it mean the last version doesn't return an exception? In which case, shall we duplicate `error.type` in `error.msg` (I don't see where we show error.type in the UI)

Reply:
From the PR description 🙂  

> Another change is that exceptions are generally not thrown by the method, instead the method returns an error enum, and a different API throws an exception. So with 7.4.1, we check for a non-success enum, and if found, set error: 1 and error.type: KeyNotFound, where KeyNotFound is the ToString of the response enum.

But yes, maybe setting it on `error.msg` makes more sense 👍

Reply:
: facepalm: I'm a slow reviewer, it spans too much time between the time I read the description and the time I finish the review so I forget. 👴 :p